### PR TITLE
Bug fix: add RBAC check on Set Traffic Distribution button

### PR DESCRIPTION
**Pull request by vikram-raj**
Bug - https://jira.coreos.com/browse/ODC-2262

Peek 2019-11-15 03-45

### CODE REVIEW

File: frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx [hunk 1]
```diff
@@ -44,34 +48,44 @@ const RevisionsOverviewListItem: React.FC<RevisionsOverviewListItemProps> = ({
...
+    namespace: service.metadata.namespace,
+    verb: 'update',
+  });
+
+  return (
+    <>
+      <SidebarSectionHeading text="Revisions" className="revision-overview-list">
+        {/* add extra check, if sidebar is opened for a knative deployment */}
+        {canSetTrafficDistribution &&
+          (service.kind === ServiceModel.kind && (
```
Comment (christianvogt) [inline frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx]: I know your didn't add this but why is this here? When will a service not have the right kind? And when do we open this sidebar for a knative deployment?

Comment (christianvogt) [reply frontend/packages/knative-plugin/src/components/overview/RevisionsOverviewList.tsx]: Captured as tech debt item as to not hold this up: https://jira.coreos.com/browse/ODC-2304
cc @invincibleJai

**Comment by spadgett** [comment]
/hold
needs rebase

**Comment by vikram-raj** [comment]
rebased

/hold cancel
